app/src/main/java/ch/sbb/polarion/extension/generic/properties/ExtendedProperties.java

```diff
@@ -1,59 +1,42 @@
 package ch.sbb.polarion.extension.generic.properties;
 
+import lombok.EqualsAndHashCode;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import java.util.HashMap;
-import java.util.Properties;
+import java.util.Set;
 
-public class ExtendedProperties extends Properties {
+@EqualsAndHashCode
+public class ExtendedProperties {
```

Comment:
Would you like to change the name as well, for example ConfigurationProperties or ConfigParams?